network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_card.html

```diff
@@ -12,18 +12,20 @@
 each blog page's dominant tag)
 
 {% block tags %}
-  {% if hide_classifiers != True %}
-    {% with topic=page.specific.topics.first %}
+  {% with topic=page.specific.topics.first %}
```

Comment:
Lines 15 - 23 are marked as changed because I removed an if statement, as according to figma we are now showing the topic and date on all blog related pages. 

I also reformatted the code a bit